### ISSUE TITLE
Keyboard shortcuts working only after click on timeline or play button

### ISSUE THREAD

**Issue by spider-x** [bigbluebutton/bbb-playback#92]
Hello, the keyboard shortcuts(play,pause, step forward, step backward) work only after click with mouse on timeline or play button. Click to the main video frame, webcam frame or frame under the webcam video frame don't help. I have tried it in Chrome, Opera, Edge. WIndows 10. Is it possible to make it working directly after opening window with the presentation? The problem occurs on client side.

**Comment by pedrobmarin** [comment]
Hi @spider-x ,

There might be a confusion here on what are the playback shortcuts. The ones you mentioned seem to be the regular `videoJS` shortcuts (provided by the lib we use to play the medias) and makes sense to need it's controlled medias focused for them to work. The shortcuts provided by the playback are:

- shortcuts: alt + shift
  - `fullscreen`: `K`
  - `section`: `L`
  - `slides`: skip
    - `next`: `ArrowUp`
    - `previous`: `ArrowDown`
  - `swap`: `M`
  - `thumbnails`: `N`
  - `video`: seek
    - `backward`: `ArrowLeft`
    - `forward`: `ArrowRight`

Still about videoJS shortcuts, I would count them as an extra. Maybe this issue could be something like: Add play and pause to the shortcuts; that might be a valid enhancement in my opinion.

**Comment by spider-x** [comment]
Thanks for explaining. I was using the shortcuts which you have posted here. Do you have any idea how to make it working without clicking on the timeline? Or how to create custom shortcuts? And where can i suggest this function?

**Comment by pedrobmarin** [comment]
Hmm, you were saying that the alt+shift shortcuts are only working when the media bar is focused? I got lost in this part.

If you want to give a try on adding new shortcuts (which would be really nice) you can start from https://github.com/bigbluebutton/bbb-playback/blob/b869bbe8a046a3be27fd21325721aa2ad0c3aabe/src/components/player.js#L148-L164. It's not so complicated to add new ones once you get how the pieces talk to each other.

You can suggest new shortcuts/features in a new issue, since this one is more like a bug issue.

**Comment by spider-x** [comment]
Thank you for your help. I will look at it.

**Comment by pedrobmarin** [comment]
Closing this one in favor of #95
